playbooks/Vagrantfile

```diff
@@ -0,0 +1,34 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.box_check_update = true
+
+  config.vm.network "forwarded_port", guest: 80, host: 8080
+  config.vm.network "forwarded_port", guest: 443, host: 8443
+
+  config.vm.provider "virtualbox" do |vb|
+    vb.gui = false
+    vb.memory = "1024"
+  end
+
+  # Fix the annoyance from https://github.com/mitchellh/vagrant/issues/1673
+  config.ssh.shell = "bash -c 'BASH_ENV=/etc/profile exec bash'"
+
+  # Fix a stubborn locale bug affecting PostgreSQL on Ubuntu
+  config.vm.provision :shell, :inline => <<-EOT
```

Comment:
@ngash @kmwenja @rexxor @bmarika The amount of "fun" caused by locale forwarding ( default behavior of the SSH client in recent Ubuntu desktop releases ) was just "epic".

This gives the Vagrant box some immunity from that.